modules/digitaluniverse/rendering/renderablebillboardscloud.cpp

```diff
@@ -290,11 +280,8 @@ namespace {
         // objects being rendered
         std::optional<std::string> labelFile;
```

Comment:
Would it make sense to move all of the label-related properties into a subtable instead?
Making:
```lua
  Renderable = {
    Type = "RenderableBillboardsCloud",
    Enabled = false,
    Color = { 0.4, 0.4, 0.9 },
    Opacity = 0.65,
    File = speck .. "/pn.speck",
    Texture = textures .. "/point4.png",
    PolygonSides = 3,
    LabelFile = speck .. "/pn.label",
    Unit = "pc",
    TextColor = { 0.25, 0.25, 0.65 },
    ScaleFactor = 425.0,
    TextSize = 16.24,
    TextMinMaxSize = { 4, 25 },
    BillboardMinMaxSize = { 0.0, 500.0 },
    EnablePixelSizeControl = true
  },
```
into
```lua
  Renderable = {
    Type = "RenderableBillboardsCloud",
    Enabled = false,
    Color = { 0.4, 0.4, 0.9 },
    Opacity = 0.65,
    File = speck .. "/pn.speck",
    Texture = textures .. "/point4.png",
    PolygonSides = 3,
    Unit = "pc",
    Labels = {
      File = speck .. "/pn.label",
      Color = { 0.25, 0.25, 0.65 },
      Size = 16.24,
      MinMaxSize = { 4, 25 },
    },
    ScaleFactor = 425.0,
    BillboardMinMaxSize = { 0.0, 500.0 },
    EnablePixelSizeControl = true
  },
```

Maybe not worth it, but worth considering at least

Reply:
It absolutely makes sense to group them together, and makes the asset clearer to read. Didn't put too much work into things like this to avoid breaking changes for more properties than needed, but maybe it's best to just do them all at once?

Also I thought we plan on refactoring the label code down the line to make them more generic anyways? For example, similarly to how the globes do it, we could have a "LabelsComponent" that encapsulates information like this. SO didn't know how much work I should actually put into refactoring